app/Http/Controllers/Maps/DeviceDependencyController.php

```diff
@@ -46,7 +46,7 @@ public function dependencyMap(Request $request): View
         $data = [
             'page_refresh' => Config::get('page_refresh', 300),
             'group_id' => $group_id,
-            'options' => Config::get('network_map_vis_options'),
+            'options' => Config::get('network_map_dependencymap_vis_options') ?? Config::get('network_map_vis_options'),
```

Comment:
Might be worth a quick mention in the docs for this (can you also add a bit for network_map_vis_options pls?)

Reply:
I did a big overhaul of the map documentation as part of the big update, including vis.js options here:
https://docs.librenms.org/Extensions/VisJS-Config/

I've just updated to use lnms config:set commands instead of php $config[] variables, and added something about this new option.